covalent_ssh_plugin/ssh.py

```diff
@@ -99,30 +106,43 @@ def __init__(
         self.cache_dir = str(Path(self.cache_dir).expanduser().resolve())
 
         self.run_local_on_ssh_fail = run_local_on_ssh_fail
+
+        self.remote_workdir = remote_workdir or get_config("executors.ssh.remote_workdir")
+        self.create_unique_workdir = create_unique_workdir or get_config("executors.ssh.create_unique_workdir")
```

Comment:
If the user specified `create_unique_workdir = False`, it will be overriden and pulled from the config instead.

I think the more direct approach of           `get_config("executors.ssh.create_unique_workdir") if create_unique_workdir is None else create_unique_workdir` might be more robust, assuming the default is `None`.

Reply:
done, the default value is `None` in the constructor but `False` in `_EXECUTOR_PLUGIN_DEFAULTS`

Reply:
Sweet, thanks for helping with the consistency overall! All makes sense to me!